__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require('supertest');
+const app = require('../app.js');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const data = require('../db/data/test-data');
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('/api/topics', () =>
+{
+    describe('GET', () =>
+    {
+        test("STATUS 200 - Responds with an array of topic objects with the properties: 'slug' and 'description'.", () =>
+        {
+            return request(app)
+                .get('/api/topics')
+                .expect(200)
+                .then(({ body: { topics } }) =>
+                {
+                    expect(topics).toHaveLength(3);
```

Comment:
Excellent to see that you included an assertion to ensure the array has some elements within it.